trace/HTTPAttributes.md

```diff
@@ -0,0 +1,35 @@
+# HTTP Attributes
+
+This document summarizes HTTP attributes recorded from HTTP
+requests and responses.
+
+HTTP integrations should set the following attributes on the client
+and server spans. For a server, request represents the incoming request.
+For a client, request represents the outgoing request.
+
+All attributes are optional.
+
+| Attribute name            | Description                 | Example value                   |
+|---------------------------|-----------------------------|---------------------------------|
+| "http.host"               | Request URL host            | "example.com"                   |
+| "http.method"             | Request URL method          | "GET"                           |
+| "http.path"               | Request URL path            | "/users"                        |
```

Comment:
maybe this could hit home by using a path value that corresponds to the route. Ex. "/users/25f4c31d"